core/arch/arm/kernel/kern.ld.S

```diff
@@ -261,13 +283,24 @@ SECTIONS
 		__start_phys_sdp_mem_section = . ;
 		KEEP(*(phys_sdp_mem_section))
 		__end_phys_sdp_mem_section = . ;
-		. = ALIGN(8);
+
+		/*
+		 * Insure init sections ends on mapping page
```

Comment:
This alignment is required when separating executable memory from read-only memory.
Can onoly drop it if we discard the read-only/non-executable requirement.